### PR TITLE
MARKET-122 Implement demo project

### PR DESCRIPTION
**Pull request by ivy-lmu**

### CODE REVIEW

File: excel-connector-demo/processes/ExcelConnectorDemo.mod [hunk 1]
```diff
@@ -0,0 +1,56 @@
+[Ivy]
+17CFF194E102686A 9.3.0 #module
+>Proto >Proto Collection #zClass
+Eo0 ExcelConnectorDemo Big #zClass
```
Comment (ivy-lmu) [inline excel-connector-demo/processes/ExcelConnectorDemo.mod:4]: @ivy-rew korrekt?

File: excel-connector-demo/src_hd/com/axonivy/connector/excel/demo/PersonEditor/PersonEditor.xhtml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+						value="Proceed" update="form" icon="ui-icon-check" />
+					<p:commandButton id="export" ajax="false"
+						actionListener="#{logic.export}" value="Export" update="form"
+						icon="ui-icon-cloud-download">
+						<p:fileDownload value="#{data.streamedExcelFile}" />
+					</p:commandButton>
+				</div>
+				<br />
+				<br />
+				<p:outputLabel value="#{data.persons}"></p:outputLabel>
```
Comment (ivy-rew) [inline excel-connector-demo/src_hd/com/axonivy/connector/excel/demo/PersonEditor/PersonEditor.xhtml:68]: remove prototype demo label

File: excel-connector-demo/src_hd/com/axonivy/connector/excel/demo/PersonEditor/PersonEditorProcess.mod [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+Ps0 f11 @C|.xml '<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<elementInfo>
+    <language>
+        <name>recordset mapping</name>
+    </language>
+</elementInfo>
+' #txt
+Ps0 f11 168 202 112 44 -52 -8 #rect
+Ps0 f12 109 224 168 224 #arcP
+Ps0 f13 processCall excel_connector/WriteExcel:call2(Recordset) #txt
```
Comment (ivy-rew) [inline excel-connector-demo/src_hd/com/axonivy/connector/excel/demo/PersonEditor/PersonEditorProcess.mod:127]: call2 seems to generic for a public interface

File: excel-connector-demo/src_hd/com/axonivy/connector/excel/demo/PersonEditor/PersonEditorProcess.mod [hunk 4]
```diff
@@ -0,0 +1,270 @@
...
+</elementInfo>
+' #txt
+Ps0 f29 144 394 160 44 -75 -8 #rect
+Ps0 f30 304 416 360 416 #arcP
+Ps0 f31 actionTable 'out=in;
+' #txt
+Ps0 f31 actionCode 'import com.axonivy.connector.excel.demo.Person;
+
+for(Record r : in.importRecordset.getRecords()){
+	Person p = new Person();
```
Comment (ivy-rew) [inline excel-connector-demo/src_hd/com/axonivy/connector/excel/demo/PersonEditor/PersonEditorProcess.mod:207]: next task: can we implement it generically ... without referring 'person' fields?

**Review by ivy-rew — APPROVED** [review]
thanks @ivy-lmu

**Pull request merged**